src/explore-education-statistics-frontend/.env.dev01

```diff
@@ -3,3 +3,5 @@ DATA_API_BASE_URL=https://s101d01-as-ees-data.azurewebsites.net
 FUNCTION_API_BASE_URL=https://s101d01-fa-ees-notify.azurewebsites.net
 GA_TRACKING_ID=UA-139424173-2
 GA_TRACKING=true
+HOTJAR_TRACKING=true
```

Comment:
Ideally we only want this enabled on live so as to not skew any stats with our local development / testing habbits. But happy to merge this in to allow us to test it provided we revert this before marking as complete

Reply:
Yeah just for testing purposes in the first instance.